rpcserver.go

```diff
@@ -543,7 +543,7 @@ func handleCreateRawTransaction(s *rpcServer, cmd interface{}, closeChan <-chan
 	params := s.cfg.ChainParams
 	for encodedAddr, amount := range c.Amounts {
 		// Ensure amount is in the valid range for monetary amounts.
-		if amount <= 0 || amount > btcutil.MaxSatoshi {
+		if amount <= 0 || amount*btcutil.SatoshiPerBitcoin > btcutil.MaxSatoshi {
```

Comment:
Could you please add a unit-test?

Reply:
I'm using google translation so I'm sorry if it's strange English.

There is no test that calls `handleCreateRawTransaction`, so does it mean I make `rpcserver_test.go`?
Even if added to `btcjson/chainsvrcmds_test.go`, this lines will not be tested.

Reply:
Ah, you're right. Too bad we don't have rpcserver tests. I tested your changes and it looks good. ✔️ 

```sh
$ btcctl --regtest createrawtransaction "[{\"txid\":\"ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\",\"vout\":0}]" "{\"mtXWDB6k5yC5v7TcwKZHB89SUp85yCKshy\": 21000000}"
0100000001ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff0000000000ffffffff010040075af07507001976a9148eb446f809f526fb37059a32cf8255c4cb43d2da88ac00000000

$ btcctl --regtest createrawtransaction "[{\"txid\":\"ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\",\"vout\":0}]" "{\"mtXWDB6k5yC5v7TcwKZHB89SUp85yCKshy\": 21000001}"
-3: Invalid amount
```

Can you please rebase your branch on the latest `master`?

Reply:
Thanks review.
This branch was created in 1db1b6f8217f5b489505c5c7e5235b53a7484779 (latest commit of master branch).
I don't think it needs to be changed.

I will rebase if needed.

Reply:
Oops, sorry I was checking against the wrong fork. Doesn't need a rebase. 👍